### PR TITLE
Add Docker image for console

### PR DESCRIPTION
**Pull request by philipsens**
Also added an empty version, so IntelliJ recognizes the file as Docker compose.
Furthermore, reformatted the spacing and added project names so it is more obvious in the log.

### CODE REVIEW

**Review by nielsm5 — APPROVED** [review]

File: docker/activemq.yml [hunk 1]
```diff
@@ -1,12 +1,14 @@
+version: ''
```
Comment (jkosternl) [inline docker/activemq.yml:1]: Please remove the newly added `version` properties. Docker should warn you about it. At my machine it outputs: "`version` is obsolete"

**Review by jkosternl — CHANGES_REQUESTED** [review]
Can you also revert the formatting everywhere? It is harder to see the line changes in Git, and it does not really improve a lot.

**Comment by philipsens** [comment]
> Can you also revert the formatting everywhere? It is harder to see the line changes in Git, and it does not really improve a lot.

Some files had 2 spaces other files had tabs, I just made it consistent. In a GIT diff it is possible to ignore whitespace changes, so it is not needed to revert.

File: docker/activemq.yml (hunk 1 above)
Comment (philipsens) [reply docker/activemq.yml:1]: It is, but some IDE's (including IntelliJ) still use this to check if it is a Dockerfile. That's why I added it, but kept it empty.

Comment (jkosternl) [reply docker/activemq.yml:1]: I didn't have issues because of that. Are you using the latest IntelliJ version and latest plugin versions?

**Comment by jkosternl** [comment]
> > Can you also revert the formatting everywhere? It is harder to see the line changes in Git, and it does not really improve a lot.
> 
> Some files had 2 spaces other files had tabs, I just made it consistent. In a GIT diff it is possible to ignore whitespace changes, so it is not needed to revert.

Yes, I already did. But in an editor you can use the blame function to see why and when a line it changed. That's messed up because of these changes. For me, it is fine to leave it a bit inconsistent, given the disadvantage.

**Comment by philipsens** [comment]
> > > Can you also revert the formatting everywhere? It is harder to see the line changes in Git, and it does not really improve a lot.
> > 
> > 
> > Some files had 2 spaces other files had tabs, I just made it consistent. In a GIT diff it is possible to ignore whitespace changes, so it is not needed to revert.
> 
> Yes, I already did. But in an editor you can use the blame function to see why and when a line it changed. That's messed up because of these changes. For me, it is fine to leave it a bit inconsistent, given the disadvantage.

Blame also has an option to ignore whitespace changes

**Review by jkosternl — APPROVED** [review]

**Comment by jkosternl** [comment]
> Blame also has an option to ignore whitespace changes

Ok, didn't know that.

File: docker/activemq.yml (hunk 1 above)
Comment (philipsens) [reply docker/activemq.yml:1]: Oh, that's interesting, I'd like to troubleshoot that a bit.

I have version `Build #IU-241.15989.150, built on April 29, 2024` of IntelliJ and `bundled 241.15989.150` for the Docker plugin.

Comment (philipsens) [reply docker/activemq.yml:1]: That's not the newest, I'll update

Comment (philipsens) [reply docker/activemq.yml:1]: It wasn't fiexed

**Pull request merged**